packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initEditUserModalMixin.ts

```diff
@@ -1,18 +1,20 @@
+import {
+  ButtonDriver,
+  ModalDriver,
+  MultiSelectDriver,
+  TextFieldDriver,
+} from '@descope/sdk-component-drivers';
+import {
+  compose,
+  createSingletonMixin,
+  createTemplate,
+} from '@descope/sdk-helpers';
+import { formMixin, loggerMixin, modalMixin } from '@descope/sdk-mixins';
 import parsePhone from 'libphonenumber-js/min';
```

Comment:
I think parsePhone should be in sdk-helpers, since we're probably going to use it in User Profile widget.

Reply:
It's an external lib
there is no problem to use in on other packages
Do you see any reason to export it from our packages?

Reply:
No, but perhaps the `formatPhoneNumber` in `initEditModalMixin` along with the external lib should be extracted to helpers.

Reply:
Not critical at all, BTW. Can be done later as well.